test/test_torch.py

```diff
@@ -3315,6 +3319,28 @@ def rand_tensor(size, dtype, device):
             out = torch.addcmul(a, b, c, value=-1)
             self.assertTrue(not (out.isnan() or out.isinf()))
 
+    def test_addcmul_cpu_scalars(self, device):
+        # Logic is dtype agnostic, so dtype isn't tested
+        alpha = 0.5
+
+        a = torch.rand((2, 2), device=device)
+        b = torch.rand((2, 2), device=device)
+        c = torch.rand((2, 2), device=device)
+        scalar = torch.rand([], device="cpu")
+
+        if device == "cpu":
+            self.assertEqual(torch.addcmul(a, scalar, c, value=alpha),
+                             a + scalar * c * alpha)
+            self.assertEqual(torch.addcmul(a, scalar, c, value=alpha),
```

Comment:
this is the same as 3332, likely unintentional?

Reply:
omg I'm sorry you're right that's just a copy and paste issue. I'll try and be more mindful